app/serializers/base_serializer.py

```diff
@@ -1,18 +1,4 @@
-from re import sub
-
-
-def field(name, getter: None):
-    assert getter is not None
-
-    return (name, getter)
-
-
-def fields(*names):
-    data = []
-    for name in names:
-        data.append(field(name, getter=lambda model, name=name: getattr(model, name)))
-
-    return data
+import re
```

Comment:
When you are only using a single function from a module, how do you decide if you want to do import that function or import the entire module? Ie. Why are you making this change? I want to know so I know what the correct style/methodology is.

Reply:
As far as I know ... python imports/loads the entire module either way? If that's not the case I'll absolutely swap it back.

I did it for the readability; i.e. `re.sub` was more clear to me than just `sub` on its own.

Reply:
Ah ok. So when I do machine learning, the modules are super long/complex so if you dont import specific modules all your code looks like: sklearn.model_selection.train_test_split(x) and it gets hard to read, so I'm used to importing just `train_test_split`. I read about it and it seems like if you are worried about naming conflicts then do `import thing`, but if you are more worried about your code being messy and long then do `from thing import specific`. So pure convention. I'm fine then with `import thing` since our code is more likely to worry about naming oberwrites than being long and messy.